modules/freertos/10.5.1/presubmit.yml

```diff
@@ -0,0 +1,17 @@
+matrix:
+  platform:
+  - debian10
+  - ubuntu2004
+  - macos
+  - macos_arm64
+  - windows
+  bazel:
+  - 7.x
+  - 6.x
+tasks:
+  verify_targets:
+    name: Verify build targets
+    platform: ${{ platform }}
+    bazel: ${{ bazel }}
+    build_targets:
+    - '@freertos:freertos'
```

Comment:
```suggestion
    - '@freertos'
```

Reply:
This turned out to be a little more involved, because FreeRTOS requires configuration, and this kernel repo actually doesn't contain an example configuration file. So, I added one in a patch. Also added support for a POSIX port of FreeRTOS, so that we can actually build something here for the host platform.